scripts/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+mongo mongodb://localhost:27017/codelists <<EOF
+db.dropDatabase();
+EOF
+
+scriptDir=$( dirname "${BASH_SOURCE[0]}" )
+mongoimport --db codelists --collection codelists --file $scriptDir/codelists.json
+mongoimport --db codelists --collection codes --file $scriptDir/codes.json
```

Comment:
You need to include an index on the codes collection for a find by `code` and `code_list`, as such:

`db.codes.ensureIndex({"code":1,"code_list":1},{"background":true});`